mindmeld/components/question_answerer.py

```diff
@@ -251,6 +253,10 @@ def transform(doc):
 
         docs = _doc_generator(data_file)
         docs_count = _doc_count(data_file)
+
+        if clean:
+            delete_index(app_namespace, index_name, es_host, es_client)
```

Comment:
do we want to handle the error when the specified index does not exist and the flag `clean=True` is specified? guess we can have a warning message and proceed with the index creation?

Reply:
`delete_index()` will display an error if no such index exists.

Reply:
though it would raise an ValueError and then load_kb() would error out?

Reply:
Shouldn't we do the same with the entity resolver's fit(clean=True) as well? https://github.com/cisco/mindmeld/blob/ea2d1e3e2b9b38ae13c63de8af4d93cf3ab2c3d8/mindmeld/components/entity_resolver.py#L167